tutorials/tutorial2.ipynb

```diff
@@ -197,178 +197,51 @@
-      "INFO:2020-06-10 16:30:32,494:AtacWorks-bw2h5] batch 0 of 58\n",
-      "INFO:2020-06-10 16:31:40,837:AtacWorks-bw2h5] batch 10 of 58\n",
-      "INFO:2020-06-10 16:32:45,415:AtacWorks-bw2h5] batch 20 of 58\n",
-      "INFO:2020-06-10 16:33:50,764:AtacWorks-bw2h5] batch 30 of 58\n",
-      "INFO:2020-06-10 16:34:55,029:AtacWorks-bw2h5] batch 40 of 58\n",
-      "INFO:2020-06-10 16:36:04,497:AtacWorks-bw2h5] batch 50 of 58\n",
-      "INFO:2020-06-10 16:36:54,095:AtacWorks-bw2h5] Done! Saved to ./NK.50_cells.h5\n"
-     ]
-    }
-   ],
-   "source": [
-    "!python $atacworks/scripts/bw2h5.py \\\n",
-    "           --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \\\n",
-    "           --intervals intervals/hg19.50000.genome_intervals.bed \\\n",
-    "           --out_dir ./ \\\n",
-    "           --prefix NK.50_cells \\\n",
-    "           --pad 5000 \\\n",
-    "           --nolabel"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This creates a file `NK.50_cells.h5`, which contains the noisy ATAC-seq signal to be fed to the pre-trained model.\n",
-    "\n",
-    "For more information type `python $atacworks/scripts/bw2h5.py --help`"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Step 7: Inference on selected intervals, producing denoised track and binary peak calls"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "INFO:2020-06-10 16:36:56,424:AtacWorks-main] Checkng input files for compatibility\n",
-      "\u001b[33mBuilding model: resnet ...\u001b[0m\n",
-      "\u001b[33mLoading model weights from ./models/model.pth.tar...\u001b[0m\n",
-      "\u001b[33mFinished loading.\u001b[0m\n",
-      "INFO:2020-06-10 16:37:06,095:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "INFO:2020-06-10 16:37:06,181:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "INFO:2020-06-10 16:37:06,231:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "INFO:2020-06-10 16:37:06,232:AtacWorks-model_utils] Compiling model in DistributedDataParallel\n",
-      "\u001b[33mFinished building.\u001b[0m\n",
-      "Inference -------------------- [ 0/29] \n",
-      "\u001b[33mInference time taken:  189.338s (Load    9.037s,Prediction  179.972s)\u001b[0m\n",
-      "INFO:2020-06-10 16:40:22,944:AtacWorks-main] Waiting for writer to finish...\n",
-      "Writing the output to bigwig files\n",
-      "Saving config file to ./infer_config.yaml...\u001b[0m\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "!atacworks denoise \\\n",
-    "    --input_files NK.50_cells.h5 \\\n",
+    "    --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \\\n",
+    "    --interval_size 50000 \\\n",
```

Comment:
Shouldn't the config files be supplied with this command?

Reply:
Not needed for the tutorial. It's needed if the user makes any custom changes. Same as training.